cdap-docs/admin-manual/source/system-requirements.rst

```diff
@@ -99,7 +99,7 @@ You'll need this software installed:
 
 Java Runtime
 ------------
-The latest `JDK or JRE version 1.7.xx or 1.8.xx <http://www.java.com/en/download/manual.jsp>`__
+The latest `JDK or JRE version 1.8.xx <http://www.java.com/en/download/manual.jsp>`__
```

Comment:
Aren't there after 1.8.x that we support? Like, Java 10.

Reply:
Ah nevermind; we don't currently support Java 10:
https://issues.cask.co/browse/CDAP-14312